sdk/eventhub/eventhubs-checkpointstore-blob/package.json

```diff
@@ -63,7 +63,7 @@
   },
   "dependencies": {
     "@azure/event-hubs": "5.0.0-preview.4",
-    "@azure/storage-blob": "12.0.0-preview.3",
+    "@azure/storage-blob": "12.0.0-preview.4",
```

Comment:
This should be done in master branch

Reply:
And can only be done after we release 12.0.0-preview.4 since it is not avaliable in master

Reply:
Yes

Reply:
@chradek after we release storage preview.4 this change will need to be made in master branch in order to use the latest version.